resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/web/ResourceController.java

```diff
@@ -128,7 +128,7 @@ private Boolean addResourceRequest(final AddResourceRequest request) {
             return service.addResource(request).get();
         } catch (InterruptedException | ExecutionException e) {
             LOGGER.error("Error while handling request {}: encountered {} {}", request, e.getClass(), e.getMessage());
-            return null;
+            return false;
```

Comment:
Why does this method return false on error while all the other methods still return null on error?

Reply:
reverted back to return null instead.